README.md

```diff
@@ -0,0 +1,198 @@
+Classy Test
+===========
+
+Meteor package which provides a class-based wrapper around `tinytest`.
+
+It has the following features:
+
+ * Class-based test cases.
+ * Common setUp/tearDown methods for test cases, separated for server and client side.
+ * A single test can interleave client-side and server-side assertions.
+ * Support for asynchronous tests.
+ * Compatible with tinytest (all tests are actually registered via tinytest).
+
+Installation
+------------
+
+```
+meteor add peerlibrary:classy-test
+```
+
+Test cases
+----------
+
+Each test case is a class extending from `ClassyTestCase` as follows:
+
+```coffeescript
+class SimpleTestCase extends ClassyTestCase
+  testThatTrueIsTrue: =>
+    @assertTrue true, "True should be true."
+
+  testThatFalseIsFalse: =>
+    @assertFalse false, "False should be false."
+
+# Register the test case.
+ClassyTestCase.addTest new SimpleTestCase()
+```
+
+This simple test case definition will generate two tests via tinytest, the first will be called `SimpleTestCase - ThatTrueIsTrue` and the second one `SimpleTestCase - ThatFalseIsFalse`.
+
+Assertions
+----------
+
+Classy test assertions are named slightly differently than in tinytest, but are otherwise equivalent with some additional assertions provided by default. Because test cases are class-based, assertions are methods which may be called in test context. The list of assertions is as follows:
+
+ * `assertEqual(a, b, message)` asserts that `a` is equal to `b`.
+ * `assertNotEqual(a, b, message)` asserts that `a` is not equal to `b`.
+ * `assertInstanceOf(object, class)` asserts that `object` is an instance of `class`.
+ * `assertMatches(string, regexp, message)` asserts that `string` matches the regular expression `regexp`.
+ * `assertThrows(function, exception)` asserts that `function` throws `exception`.
+ * `assertTrue(value, message)` asserts that `value` is `true`.
+ * `assertFalse(value, message)` asserts that `value` is `false`.
+ * `assertIsNull(value, message)` asserts that `value` is `null`.
+ * `assertIsNotNull(value, message)` asserts that `value` is not `null`.
+ * `assertIsUndefined(value, message)` asserts that `value` is `undefined`.
+ * `assertIsNaN(value, message)` asserts that `value` is `NaN`.
+ * `assertContains(sequence, value)` asserts that `sequence` contains an element `value`.
```

Comment:
One which is really useful is to test if two lists are equal no matter the order. See [implementation here](https://github.com/peerlibrary/meteor-peerdb/blob/master/tests.coffee#L300).

BTW, should we also try and port PeerDB tests to this? This could be a good test of tests. :-)